libs/core/package.json

```diff
@@ -24,6 +24,8 @@
       "types": "./dist/types/index.d.ts",
       "default": "./dist/index.js"
     },
+    "./scss/global-styles": "./scss/_global-styles.scss",
+    "./scss/utils": "./scss/_utils.scss",
```

Comment:
Could also be wildcard export to avoid an error in the future, but that would mean we expose all our scss partials as part of our public package, so that would probably be a bit much 🤔

```
"./scss/*": {
  "sass": "./scss/_*.scss" 
},
```

Reply:
Yes, let's keep it as explicit exports 👍🏻